example/src/pages/handle-redirect/index.tsx

```diff
@@ -1,16 +1,20 @@
 import React from "react";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import dotenv from "dotenv";
 
 import { handleRedirect, init } from "web3modal/src/sdk";
 
 const HandleRedirect: React.FC = () => {
+  const wallyClientId = process.env.REACT_APP_WALLY_CLIENT_ID;
+
   if (typeof window !== "undefined") {
     init({
-      clientId: "15672a04-5ce6-48ff-991c-54ab200bdd5b",
+      clientId: wallyClientId!
     });
```

Comment:
do you need to `init` again here? you should really only need one call to `init` in the main app wrapper, and then you don't need to pass the `clientId` around too